Resources/public/js/views/fields/ez-relation-editview.js

```diff
@@ -91,16 +91,51 @@ YUI.add('ez-relation-editview', function (Y) {
          * @return Object
          */
         _variables: function () {
-            var dest = this.get('destinationContent');
+            var destinationContent = this.get('destinationContent'),
+                destinationContentJSON = null,
+                isLoaded = false;
+
+            if (destinationContent !== null) {
+                isLoaded = true;
```

Comment:
It would be clearer to write:
```
isLoaded = destinationContent !== null;

if (!isLoaded && (this._isNewRelation(destinationContent) || !this._isTrashed(destinationContent))) {
      destinationContentJSON = destinationContent.toJSON();
}
// ...
```

Reply:
And even assign this:
`this._isNewRelation(destinationContent) || !this._isTrashed(destinationContent`
to some variable to let others know, what exactly this condition express.